ticdc/troubleshoot-ticdc.md

```diff
@@ -237,3 +237,7 @@ Open protocol 的输出中 type = 6 即为 null，比如：
 | Null        | 6    | `{"t":6,"v":null}` | |
 
 更多信息请参考 [Open protocol Event 格式定义](/ticdc/ticdc-open-protocol.md#column-的类型码)。
+
+## TiCDC 启动任务 start-ts 与当前时间差距较大，运行过程中同步中断，出现错误 [CDC:ErrBufferReachLimit]
+
+在 4.0.9 之后可以尝试开启 unified sorter 特性进行同步；或者可以使用 BR 工具进行一次增量备份和恢复，然后从新的时间点开启 TiCDC 同步任务。TiCDC 将会在后续版本进行优化
```

Comment:
Please add a link about the feature.

Reply:
https://github.com/pingcap/docs-cn/pull/4959 is not merged yet.